### DISCUSSION TITLE
Estimate AWS, Azure and GCP footprint with onpremises data

### DISCUSSION THREAD

**Discussion by DiegoDeMauro** [cloud-carbon-footprint/cloud-carbon-footprint#1283] [Ideas]
Was wondering how can I estimate Cloud footprint with the onpremises csv. 

Thats particularly usefull to estimate the footprint reduction after migrating to the cloud.

**Comment by 4upz** [comment]
Hi @DiegoDeMauro-Navigator it sounds like you are trying to estimate cloud usage based on similar on-premise workloads. We don't have direct support for this, but you may be able to use our [Lookup Table](https://www.cloudcarbonfootprint.org/docs/creating-a-lookup-table/) feature in combination with a cloud provider's pricing calculator to map out similar services and input them as line items into the Lookup Table. The Lookup Table will estimate energy and carbon for a single unit of usage for each service, where you can play around with the usage amount to see how your current workload would compare. Feel free to let me know whether that makes sense!

**Reply by DiegoDeMauro** [reply]
Thanks, that would work for me. We are building our own dashboard to forecast carbon footprint on the various cloud providers if migrating on premises.
